devel/libwebsockets/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; indent-tabs-mode: nil -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        warmcat libwebsockets 2.4.1 v
+categories          devel net
+platforms           darwin
+license             LPGL-2.1
+
+maintainers         nomaintainer
```

Comment:
Are you willing to take over the maintainership?

Reply:
I am willing, but does maintainer require MacPorts account? That I don't have.

Reply:
No. You start by claiming some ports. Once you get sufficient expertise and understanding of the system, you can request commit rights, but that's not a requirement for being a maintainer. It helps if you are the maintainer because the changes will probably get merged faster and any bugs or feature requests will be assigned to you. In that case please provide both obfuscated form of your email and your github handle.